source/manual/nagstamon.html.md

```diff
@@ -1,7 +1,7 @@
 ---
 owner_slack: "@christopherbaines"
```

Comment:
is Chris still the best person for this? Or should we move it to govuk-developers?

Reply:
Does anyone but Chris use Nagstamon?

Reply:
I do....very occasionally when I'm on 2ndline and WFH.....but yeah we could leave it for now. I'm sure Chris will updated it if it becomes a burden